api/bin/execute_sql_rds.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#
+# Execute some SQL using the RDS Data API.
+#
+# Examples:
+#   ./execute_sql_rds.sh <sql/table_list.sql
+#   ./execute_sql_rds.sh --multiple <sql/select_from_foreign_table.sql
+#
+# When using --multiple, provide one SQL statement per input line.
+#
+
+set -o errexit -o pipefail
+
+PROGRAM_NAME=$(basename "$0")
+
+CYAN='\033[96m'
+GREEN='\033[92m'
+END='\033[0m'
+
+RESOURCE_ARN='arn:aws:rds:...'
+SECRET_ARN='arn:aws:secretsmanager:...'
```

Comment:
I'm not sure if it would be an issue, but what about making these env vars (or params passed in) so someone is less likely to accidentally check in the ARNs?

Also - it would be good to know what user / role access a person has with this - can this only be the root user? Might be a problem we address later with our own login roles and what they have access to use (ie. almost anyone with AWS access can use a read-only role, only admins can write)